app/main.py

```diff
@@ -1,3 +1,71 @@
+from typing import Union
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: Union[int, float]) -> None:
```

Comment:
From python 3.10+ we can use `|` in annotations.

```suggestion
    def __init__(self, km: int | float) -> None:
```
Fix in all cases